tests/target/issue-6333.rs

```diff
@@ -0,0 +1,22 @@
+pub use crate::r#_StructUtil_Compile::CanonCryptoItem;
```

Comment:
Would probably be useful to add the `// rustfmt-style_edition: 2015` comment to the start of the file. Also, it might be useful to include a separate test for `// rustfmt-style_edition: 2024`.

Reply:
Done.